### PR TITLE
Added travis ci support for modern node versions

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js:
+  - "6"
```
Comment (twolfson) [inline .travis.yml]: Can we also add "5" and drop "iojs" from this list?

**Comment by amilajack** [comment]
Sure thing! Fixed.

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - "iojs"
+  - 6
+  - 5
+  - 4
+  - 0.12
+  - 0.11
+  - 0.10
```
Comment (twolfson) [inline .travis.yml]: You stripped away the quotes, please keep them for reducing the noise in this PR =/

**Comment by twolfson** [comment]
Looks like too much was fixed in that last commit. Left another comment. Can you please squash the commits into 1 commit as well?

**Comment by amilajack** [comment]
I haven't squashed a commit before. How would I do that?

**Comment by twolfson** [comment]
Run `git rebase -i master` and change all commits except for the first 1 to "squash". Here's a demonstration:

https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

Once the commits are squashed into 1, you will need to force push the update. That can be done via:

```bash
git push origin patch-1 --force
```

**Comment by dawsonbotsford** [comment]
Why don't we have node v6 being tested in this PR? Just joining this discussion.

**Comment by twolfson** [comment]
It is in this PR, maybe you are viewing a subset of the commits?

https://github.com/uber/image-diff/pull/41/files#diff-354f30a63fb0907d4ad57269548329e3R3

https://travis-ci.org/uber/image-diff/jobs/145087157

**Comment by amilajack** [comment]
The command
```
git push origin patch-1 --force
```
isnt working for me. It's returning `Everything up-to-date` but the squash doesnt seem to be pushed.

**Comment by twolfson** [comment]
You need to run the `git rebase -i master` first and choose "squash" for all but the first commit as instructed =/

**Comment by twolfson** [comment]
Also, make sure you are on the `patch-1` branch when running `git rebase`

**Comment by amilajack** [comment]
I see. Can I close this PR and make a new one with these revisions?

**Comment by twolfson** [comment]
You can but you should be able to do everything on this branch/PR =/

**Pull request closed without merging**